padguidedb/padguidedb.py

```diff
@@ -157,6 +162,24 @@ async def dungeondrops(self, ctx, dungeon_id: int, dungeon_floor_id: int):
             for page in pagify(msg):
                 await ctx.send(box(page))
 
+    @padguidedb.command()
+    @is_padguidedb_admin()
+    async def dungeondata(self, ctx, dungeon_id: int):
+        with ctx.typing(), self.get_connection() as cursor:
+            sql = "SELECT * FROM wave_data WHERE dungeon_id = {}".format(dungeon_id)
+            cursor.execute(sql)
+            results = list(cursor.fetchall())
```

Comment:
You could literally crash miru and/or random processeses on the server (via OOM killer) with this. Take the function out of the PR for now and we can figure out what to do with it later.